dist/v-mapbox.umd.js missing from published package

The `main` key in the package.json is set to `dist/v-mapbox.umd.js`. However, that file does not exist, leading to:

```
 Cannot find module 'v-mapbox' from '/path/to/project' 
```

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "1.6.2",
   "private": false,
   "sideEffects": false,
-  "main": "dist/v-mapbox.umd.js",
+  "main": "dist/v-mapbox.js",
   "module": "dist/v-mapbox.esm.js",
   "unpkg": "dist/v-mapbox.min.js",
   "jsdelivr": "dist/v-mapbox.min.js",

